src/Output.cpp

```diff
@@ -0,0 +1,109 @@
+#include "CodeGen_C.h"
+#include "StmtToHtml.h"
+#include "Output.h"
+#include "LLVM_Headers.h"
+#include "LLVM_Output.h"
+
+#include <fstream>
+
+namespace Halide {
+
+void output_object(const Module &module, std::string filename) {
```

Comment:
"output" is a very generic name. "compile_to" seems like a better prefix (ala the Func methods), but that seems weird for text or html. Any better ideas?